<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KlientDialog</class>
 <widget class="QDialog" name="KlientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>House *</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="TelefonEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="UlicaEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>Client name *</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="MiastoEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>City *</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Street *</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="MailEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="NazwaEdit"/>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="NumerEdit">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2500</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Phone number</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Mail</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>NazwaEdit</tabstop>
  <tabstop>TelefonEdit</tabstop>
  <tabstop>MailEdit</tabstop>
  <tabstop>MiastoEdit</tabstop>
  <tabstop>UlicaEdit</tabstop>
  <tabstop>NumerEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KlientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KlientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NazwaEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>KlientDialog</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UlicaEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>KlientDialog</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MiastoEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>KlientDialog</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NumerEdit</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>KlientDialog</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTextChanged()</slot>
 </slots>
</ui>
